picoscope/ps4000a.py

```diff
@@ -278,28 +283,68 @@ def _lowLevelGetTimebase(self, tb, noSamples, oversample, segmentIndex):
 
     def getTimeBaseNum(self, sampleTimeS):
         """Return sample time in seconds to timebase as int for API calls."""
-        maxSampleTime = (((2 ** 32 - 1) - 4) / 2e7)
 
-        if sampleTimeS <= 12.5E-9:
-            timebase = math.floor(math.log(sampleTimeS * 8E7, 2))
-            timebase = max(timebase, 0)
-        else:
-            # Otherwise in range 2^32-1
-            if sampleTimeS > maxSampleTime:
-                sampleTimeS = maxSampleTime
+        if self.model in ('4828'):
```

Comment:
I think this should be `if self.model in ('4828',):`, notice the comma after the `'4828'`

Reply:
or why not even `self.model == '4828'` ???

Reply:
Right, just let those fail. I've thought about making this more generated in the past, but I feel like there are enough small changes in the API and the dtypes that it isn't worth it.